{"hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics","doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics\/Fortify"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"text":"func","kind":"keyword"},{"text":" ","kind":"text"},{"text":"escape","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"withError"},{"kind":"text","text":" "},{"text":"error","kind":"internalParam"},{"text":": any ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") -> "},{"text":"Never","kind":"typeIdentifier","preciseIdentifier":"s:s5NeverO"}],"platforms":["macOS"],"languages":["swift"]}]},{"kind":"parameters","parameters":[{"name":"error","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"Object conforming to Error."}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"identifier":{"url":"doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics\/Fortify\/escape(withError:)","interfaceLanguage":"swift"},"metadata":{"role":"symbol","roleHeading":"Type Method","externalID":"s:7Swallow7FortifyO6escape9withErrors5NeverOs0E0_p_tFZ","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"func","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"escape"},{"kind":"text","text":"("},{"text":"withError","kind":"externalParam"},{"kind":"text","text":": any "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5NeverO","text":"Never"}],"modules":[{"name":"Diagnostics"}],"symbolKind":"method","title":"escape(withError:)"},"kind":"symbol","abstract":[{"type":"text","text":"Escape from current execution context, rewind the stack"},{"type":"text","text":" "},{"text":"and throw error from the last time protect was called.","type":"text"}],"variants":[{"paths":["\/documentation\/diagnostics\/fortify\/escape(witherror:)"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"references":{"doc://com.swallow.documentation.Diagnostics/documentation/Diagnostics/Fortify/escape(withError:)":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"kind":"identifier","text":"escape"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"withError"},{"kind":"text","text":": any "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":") -> ","kind":"text"},{"text":"Never","kind":"typeIdentifier","preciseIdentifier":"s:s5NeverO"}],"kind":"symbol","role":"symbol","title":"escape(withError:)","abstract":[{"text":"Escape from current execution context, rewind the stack","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"and throw error from the last time protect was called."}],"url":"\/documentation\/diagnostics\/fortify\/escape(witherror:)","type":"topic","identifier":"doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics\/Fortify\/escape(withError:)"},"doc://com.swallow.documentation.Diagnostics/documentation/Diagnostics/Fortify":{"abstract":[],"url":"\/documentation\/diagnostics\/fortify","navigatorTitle":[{"text":"Fortify","kind":"identifier"}],"kind":"symbol","title":"Fortify","identifier":"doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics\/Fortify","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Fortify","kind":"identifier"}],"type":"topic"},"doc://com.swallow.documentation.Diagnostics/documentation/Diagnostics":{"abstract":[],"url":"\/documentation\/diagnostics","kind":"symbol","role":"collection","type":"topic","title":"Diagnostics","identifier":"doc:\/\/com.swallow.documentation.Diagnostics\/documentation\/Diagnostics"}}}